tox.ini

```diff
@@ -21,3 +21,10 @@ deps =
 
 commands =
     pytest
+
+
+[testenv:typing]
+deps =
+    mypy
+commands =
+    mypy
```

Comment:
Would it be useful to also add mypy to CI?

Reply:
Since CI runs tox, wouldn't this be run in CI anyway?

Reply:
It won’t because you did not add it to `envlist`

Reply:
So that means that tests are also not being run (and looking at CI logs, that is the case)